Duplicati/Library/Main/Options.cs

```diff
@@ -1839,7 +1839,7 @@ public int ConcurrencyCompressors
             {
                 if (m_compressionHints == null)
                 {
-                    var hints = new Dictionary<string, CompressionHint>(Library.Utility.Utility.ClientFilenameStringComparer);
+                    var hints = new Dictionary<string, CompressionHint>(StringComparer.OrdinalIgnoreCase); // Since file extensions are case insensitive

```

Comment:
Thanks @samuel-w.  I think we can try and improve this a little bit by providing some more detail.  Perhaps something like this?
```
// Even if the filesystem is case-sensitive, we will ignore the case when examining the extension for
// compression purposes.  The casing of the extension rarely matters as a file type hint (jpg vs. JPG).
```

Reply:
```Ignore file system case sensitivity, since file extensions case rarely indicates type```